src/build-from-oxlint-config.ts

```diff
@@ -168,11 +178,48 @@ const handleRulesScope = (
       rules[eslintName] = 'off';
     } else if (rule in rules && isDeactivateValue(oxlintRules[rule])) {
       // rules extended by categories or plugins can be disabled manually
-      delete rules[eslintName];
+      if (disablesRule) {
+        delete rules[eslintName];
+      }
+      // inside overrides we need to enable the rule again
+      else {
+        rules[eslintName] = 'warn';
```

Comment:
@Boshen 

With a config like:

```
{
  "rules": {
    "no-debugger": "error",
  },
  "overrides": [
    {
      "files: "["*/.test.ts"],
      "rules": {
        "no-debugger": "off"
      }
    }
  ]
}
```

Should the rule be re enabled in `eslint` or do we want to ignore it?  
It is maybe impossible to match the user configuration from their `eslint.config.js`.
Currently I want to output it in `eslint` as a warning. But I have no idea how the user config is build.

Relevant test case in oxlint:
https://github.com/oxc-project/oxc/blob/defaf4bf2b85a011f977dfa23b14312500875555/crates/oxc_linter/src/config/flat.rs#L239-L257

Reply:
the same problem as before. Oxlint should be the main source. And we do not reenable them in eslint